.github/scripts/deploy_dbt_model_dependencies.sh

```diff
@@ -147,12 +147,12 @@ while read -r item; do
     subdirectory_name="${model_identifier}/"
     mkdir -p "$subdirectory_name"
     echo "Installing dependencies from $requirements_filename into $subdirectory_name"
-    pip install -t "$subdirectory_name" -r "$requirements_filename"
+    pip install -t "$subdirectory_name" -r "$requirements_filename" --no-deps
 
     # Create a zip archive from the contents of the subdirectory
     zip_archive_name="${model_identifier}.requirements.zip"
     echo "Creating zip archive $zip_archive_name from $subdirectory_name"
-    zip -q -r "$zip_archive_name" "$subdirectory_name"
+    cd "$subdirectory_name" && zip -q -r9 "../$zip_archive_name" ./* && cd ..
```

Comment:
This change more closely aligns us with [the recommended workflow for bundling external dependencies for Athena PySpark](https://docs.aws.amazon.com/athena/latest/ug/notebooks-import-files-libraries.html#notebooks-import-files-libraries-importing-a-python-zip-file-from-a-github-project), both by using a higher compression level for `zip` and removing the top level of the zip archive. There may be a cleaner way to implement that latter goal without requiring the `cd` calls, but in the meantime this works and is reasonably clear.